smart-contracts/Cargo.toml

```diff
@@ -39,11 +39,12 @@ dex-router-osmosis = { path = "contracts/dex-router-osmosis", features = ["libra
 lst-dex-adapter-osmosis = { path = "contracts/lst-dex-adapter-osmosis", features = ["library"] }
 lst-adapter-osmosis = { path = "contracts/lst-adapter-osmosis", features = ["library"] }
 
+
 # SDK
 cosmos-sdk-proto = {version = "0.21.1", default-features = false}
 
 # Osmosis
-osmosis-std = "0.25.0"
```

Comment:
plz keep version

Reply:
Alright let me look at cw-roch versions, it still looks to be using v24 underlying causing type mismatches

Reply:
got it, let's see if that makes it into the abstract release, else we have to go with v0.24 longer

Reply:
After we talked to abstract and discussed on slack, decided to downgrade and upgrade as soon as new version is released